Microsoft.NET.Build.Containers.IntegrationTests/ParseContainerPropertiesTests.cs

```diff
@@ -16,12 +16,13 @@ public class ParseContainerPropertiesTests
     [DockerDaemonAvailableFact]
     public void Baseline()
     {
-        var (project, _) = ProjectInitializer.InitProject(new () {
+        var (project, _, d) = ProjectInitializer.InitProject(new () {
```

Comment:
We need to dispose the ProjectCollection to ensure that the binlogs get written, again to help with debugging.